docs/section19/19-13.md

```diff
@@ -0,0 +1,75 @@
+# Section 19 - Rule 19-13             
+**Schema Version:** 0.0.24      
+**Mandatory Rule:** True    
+**Rule ID:** 19-13             
+**Rule Description:** For baseline system types 1-8 and 11-13, system design supply airflow rates shall be based on a supply-air-to-room temperature set-point difference of 20°F or the minimum outdoor airflow rate, or the airflow rate required to comply with applicable codes or accreditation standards, whichever is greater. For systems with multiple zone thermostat setpoints, use the design set point that will result in the lowest supply air cooling set point or highest supply air heating set point.  
+
+Exceptions:
+1. For baseline system types 1-8 and 11-13, for systems serving laboratory spaces, airflow rate shall be based on a supply-air-to-room temperature set-point difference of 17°F or the required ventilation air or makeup air, whichever is greater.
+2. For baseline system types 1-8, 11-13, if the proposed design HVAC system airflow rate based on latent loads is greater than the design airflow rate based on sensible loads, then the same supply-air-to-room-air humidity ratio difference (gr/lb) used to calculate the proposed design airflow shall be used to calculate design airflow rates for the baseline building design.
```

Comment:
I think exception #2 should be excluded from the RCT scope and notes as such in the body of 229P. Also, I recommend excluding references to system types 1-8 and 11-13 from the text of the exceptions. (E.g., it's weird to think that Sys 1 may serve a lab.) This would also align with 90.1 language.

Reply:
Okay, removed from this rule and is a rule 19-16 which is marked as N for evaluated.